Wrappers/Python/test/test_algorithms.py

```diff
@@ -1112,13 +1112,13 @@ def test_SPDHG_vs_PDHG_implicit(self):
 
         # block function
         F = BlockFunction(*[KullbackLeibler(b=g[i]) for i in range(subsets)])
-        G = alpha * TotalVariation(50, 1e-4, lower=0, warm_start=True)
+        G = alpha * TotalVariation(2, 1e-4, lower=0, warm_start=True)
 
         prob = [1/len(A)]*len(A)
         spdhg = SPDHG(f=F, g=G, operator=A,
                       max_iteration=1000,
                       update_objective_interval=200, prob=prob)
-        spdhg.run(1000, verbose=0)
+        spdhg.run(1000)
```

Comment:
Could probably decrease iteration number to further speed things up

Reply:
sure, but we were checking things at convergence

Reply:
Yep, but with a smaller image size I doubt we need all 1000 iterations